core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -584,7 +604,7 @@ else if (userId == null) {
      * @return List of the feature keys that are enabled for the user if the userId is empty it will
      * return Empty List.
      */
-    public List<String> getEnabledFeatures(@Nonnull String userId,@Nonnull Map<String, String> attributes) {
+    public List<String> getEnabledFeatures(@Nonnull String userId, @Nonnull Map<String, String> attributes) throws IllegalArgumentException {
```

Comment:
I don't think this should have a throws.  Any of the methods could throw an illegal argument exception because of the non null annotations.